### DISCUSSION TITLE
Releasing new version of quarkus-primefaces-extensions maven repo

### DISCUSSION THREAD

**Discussion by seklmco** [quarkusio/quarkus#41341] [Q&A] [answered]
Hello. I am unfamiliar with the process of new releases on maven. A new version of [primefaces-extensions](https://central.sonatype.com/artifact/org.primefaces.extensions/primefaces-extensions/versions), v13.0.12, was recently released. What is the process of releasing a new version of [quarkus-primefaces-extensions](https://central.sonatype.com/artifact/io.quarkiverse.primefaces/quarkus-primefaces-extensions/3.14.1/overview) to reflect the update?

[a comment by an automated account is not shown]

**Reply by mschorsch** [reply, marked as answer]
@seklmco You have to open an issue in the [quarkus-primfaces Repo](https://github.com/quarkiverse/quarkus-primefaces).
